.github/ISSUE_TEMPLATE/pg-auto-failover-enterprise-release-checklist-template.md

```diff
@@ -0,0 +1,62 @@
+---
+name: pg-auto-failover-enterprise release checklist template
+about: Please make sure to follow the checklist below along the release process.
+title: Pg Auto Failover Enterprise release checklist - [vX.Y.Z]
+labels: release_checklist
+assignees: gurkanindibay
+
+---
+
+## Several things to note here
+
+These instructions assume you have `$VERSION`, `$PROJECT`, and `$REPO` environment variables set in your shell (e.g. `10.0.2`, `citus`, and `citus`). With those set, code from most steps can be copy-pasted.
```

Comment:
I do not think that this line is correct. Is it a copy paste error?

Please go over similar lines in all templates.

Reply:
Removed these lines